### ISSUE TITLE
EPI2ME decontam Oxford nanopore sequencing 

### ISSUE THREAD

**Issue by CamillePhilippeSCI** [benjjneb/decontam#146]
Dear all users of decontam, 

I am very new to this and i am having some issues with the processing of my data, 
I have obtained a feature table using EPI2ME 16s Workflow. 
But i dont have OTU i have just the names of the bacteria in my samples, 
Would it still be possible to use decontam ? 

Thanks :) 

Camille

**Comment by benjjneb** [comment]
Yes, you would just operate on the named taxa the same way you would use OTUs or ASVs.

**Comment by CamillePhilippeSCI** [comment]
thanks !
